Resources/views/Collector/db.html.twig

```diff
@@ -25,7 +47,11 @@
         </div>
         <div class="sf-toolbar-info-piece">
             <b>Invalid entities</b>
-            <span class="sf-toolbar-status sf-toolbar-status-{{ collector.invalidEntityCount > 0 ? 'red' : 'green' }}">{{ collector.invalidEntityCount }}</span>
+            {% if profiler_markup_version == 1 %}
+                <span class="sf-toolbar-status sf-toolbar-status-{{ collector.invalidEntityCount > 0 ? 'red' : 'green' }}">{{ collector.invalidEntityCount }}</span>
+            {% else %}
+                <span class="sf-toolbar-status sf-toolbar-status-{{ collector.invalidEntityCount > 0 ? 'red' }}">{{ collector.invalidEntityCount }}</span>
```

Comment:
you are still displaying a ``sf-toolbar-status-`` class here when the condition is false, which is what I'm askign you to change since the very beginning, to produce a clean markup.

I don't care about the fact that the green background is removed in the old markup, as it is not very important here. What is important is the red background on error.
Btw, I don't think the green background for this setting is worth the duplication. I think we can live without it.